src/client/styles/fonts/_frutiger.scss

```diff
@@ -0,0 +1,7 @@
+@import url("https://fast.fonts.net/cssapi/0985eb4d-eec4-488b-baa4-22fe85c5ce32.css");
+body p, h1, h2, h3, details summary, ul {
```

Comment:
Not sure why you've been so specific with you selectors here? Why not just set it on `body`? I would be tempted to do:

```
html, body, button, input, table, td, th {
    font-family: "Frutiger W01",Arial,sans-serif;
}
```